tests/arm64/Tests.lst

```diff
@@ -1840,20 +1840,6 @@ Expected=100
 MaxAllowedDurationSeconds=600
 Categories=Pri0;EXPECTED_PASS
 HostStyle=Any
-[badldsfld_il_d.exe_263]
-RelativePath=JIT\Directed\coverage\importer\Desktop\badldsfld_il_d\badldsfld_il_d.exe
-WorkingDir=JIT\Directed\coverage\importer\Desktop\badldsfld_il_d
-Expected=100
-MaxAllowedDurationSeconds=600
-Categories=Pri0;EXPECTED_FAIL
```

Comment:
Do we really want to delete this for the Tests.lst file or just change their Categories tag to DISABLED or EXCLUDED or something similar?

Are we expecting these tests to be "fixed" or deleted in the future?